### DISCUSSION TITLE
Lesson 4 : from web3 import Web3 causes error

### DISCUSSION THREAD

**Discussion by km123abca** [smartcontractkit/full-blockchain-solidity-course-py#1036] [Q&A] [answered]
The import statement : from web3 import Web3 , results in a large error string the end of which is as shown below.

Python36-32\lib\site-packages\eth_account\hdaccount\_utils.py", line 19, in <module>
    SECP256K1_N = int("FFFFFFFF_FFFFFFFF_FFFFFFFF_FFFFFFFE_BAAEDCE6_AF48A03B_BFD25E8C_D0364141", 16)
ValueError: int string too large to convert

How do i resolve this? Pls Help

**Comment by cromewar** [comment, marked as answer]
Hello @km123abca 
For me this is because of your python version, you are using python3.6, could you use python 3.8 instead?

**Reply by km123abca** [reply]
Thanks. I was using python 3.6. Upgrading to the latest python version solved the problem.

**Reply by cromewar** [reply]
I'm glad is solved now.

**Reply by formulaRoot** [reply]
Is python 3.8 the only version that works? I was on 3.9 when i first encountered this issue, then bumped to 3.10 but still having issues.
